.flake8

```diff
@@ -0,0 +1,12 @@
+[flake8]
+max-line-length = 99
+select: E,W,F,C,N
+exclude:
```

Comment:
why do we need this "exclude" list? you're getting flake8 issues from .git dir, for example?